src/components/Identicon/Identicon.tsx

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import React from 'react';
+
+import { ComponentPropsWithoutRef, useMemo } from 'react';
+
+import { identicon } from 'minidenticons';
+
+export interface IdenticonProps extends ComponentPropsWithoutRef<'img'> {
+  username: string;
+  size?: number;
+  saturation?: number;
+  lightness?: number;
```

Comment:
Even that is optional, we'll use it!?

Reply:
maybe? for now I have no idea, UI requirement is default value saturation 50% and lightness 50%

Reply:
Oh really!? I am afraid we don't need it. Let you decide, then!